tests/unit/Correction.spec.ts

```diff
@@ -93,9 +95,9 @@ xdescribe('Correction - UI', () => {
     wrapper.destroy()
   })
 
-  it('sets filing data properly', () => {
+  it('sets filing data properly', async () => {
     // mock $route
-    const $route = { params: {} }
+    const $route = { params: { filingId: '0', correctedFilingId: '123' } }
```

Comment:
I have no idea how this worked before (without an id to fetch) !?!?

But anyway, it DIDN'T work after my changes, so I had to fix it :)